specs/eip4844/polynomial-commitments.md

```diff
@@ -293,9 +293,11 @@ def evaluate_polynomial_in_evaluation_form(polynomial: Polynomial,
 
     result = 0
     for i in range(width):
-        result += div(int(polynomial[i]) * int(roots_of_unity_brp[i]), (int(z) - int(roots_of_unity_brp[i])))
-    result = result * (pow(z, width, BLS_MODULUS) - 1) * inverse_width % BLS_MODULUS
-    return result
+        a = BLSFieldElement(int(polynomial[i]) * int(roots_of_unity_brp[i]) % BLS_MODULUS)
+        b = BLSFieldElement((int(BLS_MODULUS) + int(z) - int(roots_of_unity_brp[i])) % BLS_MODULUS)
```

Comment:
Looks good though its probably good to note the slight change here.

Before it was : `int(z) - int(roots_of_unity_brp[i])`

Now its `(int(BLS_MODULUS) + int(z) - int(roots_of_unity_brp[i])` 

When taken % BLS_MODULUS, it is the same result, since `int(BLS_MODULUS) % BLS_MODULUS = 0

The reason to do this is because `z-roots_of_unity_brp[i])` can be negative and taking that modulo BLS_MODULUS will probably give a negative result.